src/Main/Main.js

```diff
@@ -156,7 +157,7 @@ class Main extends Component {
     dropdown: {
       initialValue: {
         paymentMethodDD: [
-          { label: '-- Select --', value: '' },
+          EMPTY_DROPDOWN_ITEM,
```

Comment:
please check where it's used, it's a required field, no empty option is needed. we should handle this through `placeholder=' '` prop of `Select` field component

Reply:
I would suggest solution like in the last commit